lib/main_containers.pm

```diff
@@ -363,6 +363,7 @@ sub load_container_tests {
         } else {
             # Container Host tests
             loadtest 'microos/toolbox' if (/podman/i && !is_staging && (is_sle_micro || is_microos || is_leap_micro));
+            loadtest 'containers/enable_mac' if get_var("SECURITY_MAC");
```

Comment:
Arguably this is a generic test that should be in `console/` (or similar) and not only in `containers/`.

Reply:
Fixed.